cli/src/main/java/org/owasp/dependencycheck/App.java

```diff
@@ -302,7 +302,7 @@ private int determineReturnCode(Engine engine, float cvssFailScore) {
                         || (v.getCvssV3() != null && v.getCvssV3().getBaseScore() >= cvssFailScore)
                         || (v.getUnscoredSeverity() != null && SeverityUtil.estimateCvssV2(v.getUnscoredSeverity()) >= cvssFailScore)
                         || (cvssFailScore <= 0.0f)) { //safety net to fail on any if for some reason the above misses on 0
-                    retCode = 1;
+                    retCode = 11;
```

Comment:
```suggestion
                    retCode = 15;
```

As you intend to be able to discover 'failure for vulnerabilities' separate from other error cases the returncode should be unique and not overlap with line 196 (Database error)

Reply:
Oops! Thank you :)